### DISCUSSION TITLE
checkmate installation problem

### DISCUSSION THREAD

**Discussion by avnsh9** [CheckMATE2/checkmate2#18] [Q&A]
I am getting this error while using make command for checkmate
fatal error: classes/DelphesHepMCReader.h: No such file or directory
I have installed delphes 3.5
[make.log](https://github.com/CheckMATE2/checkmate2/files/6991774/make.log)

**Comment by physicsbamboo** [comment]
I have the same issue. Have you solved that? Thanks!
